### DISCUSSION TITLE
running ./gradlew createConfigs

### DISCUSSION THREAD

**Discussion by ikatyal2110** [TEAMMATES/teammates#12563] [Help requests] [answered]
I'm on Mac, when running "./gradlew createConfigs", I get the following error:

Welcome to Gradle 7.5!

Here are the highlights of this release:
 - Support for Java 18
 - Support for building with Groovy 4
 - Much more responsive continuous builds
 - Improved diagnostics for dependency resolution

For more details see https://docs.gradle.org/7.5/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'teammates'.
> Could not open cp_proj generic class cache for build file '/Users/ishaankatyal/Desktop/teammates/build.gradle' (/Users/ishaankatyal/.gradle/caches/7.5/scripts/10hkiewh0i2h9yuux5zldr0n2).
   > BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 64

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 20s

I don't know what the issue is or how to fix this

**Comment by wkurniawan07** [comment, marked as answer]
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' **Unsupported class file major version 64**

You were using Java 20 which is not supported.

**Reply by ikatyal2110** [reply]
thanks, I'm new to all this so just getting used to it
